src/file_search/movie/MovieDirectorySearcher.cpp

```diff
@@ -283,8 +284,27 @@ void MovieDiskLoader::createMovie(QStringList files)
         movie->controller()->loadData(Manager::instance()->mediaCenterInterface());
         if (discType == DiscType::Single) {
             QFileInfo mFi(files.first());
+
+            // Get subtitle filters from AdvancedSettings
+            const mediaelch::FileFilter& subtitleFilters = Settings::instance()->advanced()->subtitleFilters();
```

Comment:
Just FYI: I try to avoid using the settings singleton via `Settings::instance()` because it makes testing more difficult (global state and all). I never got to test this class via our tests in `test/`, but this may make it more difficult to do so.

It's fine for now. However, I will probably refactor this coding a bit and pass the "subtitleFilters" in via constructor injection. :)